### DISCUSSION TITLE
JSON Format for  Plans and Time Strips

### DISCUSSION THREAD

**Discussion by jvigliotta** [nasa/openmct#4271] [Community Q&A] [answered]
Where can the JSON format for Plans and Time Strips be found?

**Comment by shefalijoshi** [comment, marked as answer]
There is an example in the plan tests here: https://github.com/nasa/openmct/blob/master/src/plugins/plan/pluginSpec.js#L118

**Reply by Cameronseitz** [reply]
I also had this issue. To test to make sure that I can create a time strip with a valid JSON, I tried exporting the JSON from the timestrip on the demo project on openmct's website and using that file for mine, but it didn't work. It's weird because in the user guide, it says to make a Plan first and that is where you import the JSON file and then make a timestrip, but on the demo, there is no Plan, just a working timestrip. Do you know what I am doing wrong here?
